src/components/AnchorNav/index.tsx

```diff
@@ -96,9 +91,16 @@ const AnchorNav: React.FC<AnchorNavProps> = ({
     );
   };
 
+  const currentPageName = currentPage.substring(
+    currentPage.lastIndexOf("/") + 1
+  );
+
   return headings.length > 0 ? (
     <div className="side-nav">
-      <nav aria-label={`${section} section contents`} className="nav">
+      <nav
+        aria-label={`${currentPageName} ${activeId} page contents`}
```

Comment:
When testing, the screen reader does not pick this up. 

For instance here is the alert page, the screen reader should read out "alert introduction page contents" which is there when inspecting (as seen below).

<img width="356" alt="Screenshot 2024-01-19 at 10 41 45" src="https://github.com/mi6/ic-design-system/assets/111882114/e78a498e-3d90-4f50-be43-8b8ec8c63a88">

This is the output of the screen reader.
<img width="667" alt="Screenshot 2024-01-19 at 10 37 35" src="https://github.com/mi6/ic-design-system/assets/111882114/e6d22f38-d820-4b27-a3a7-dfb19536487a">

I presume this is a screen reader bug but is worth looking into to? 
Thought to raise this. 

Great work though 👍

Reply:
Yeah the aria-label describes the nav area as a whole, whereas your screenshot appears to be reading out an individual nav item.